modules/web-console/e2e/testcafe/fixtures/admin-panel.js

```diff
@@ -64,3 +72,134 @@ test('Testing setting notifications', async(t) => {
         .expect(Selector('.wch-notification', { visibilityCheck: false } ).visible)
         .notOk();
 });
+
+const _createBaseModel = async(t, keyType, valueType) => {
+    await t.click(pageAdvancedConfiguration.modelsNavButton)
+        .click(pageAdvancedModel.createModelButton)
+        .click(pageAdvancedModel.general.generatePOJOClasses.control);
+
+    await scrollToPageBottom();
+
+    await t.click(pageAdvancedModel.sqlQuery.panel.heading)
+        .typeText(pageAdvancedModel.general.keyType.control, keyType, {paste: true})
+        .typeText(pageAdvancedModel.general.valueType.control, valueType, {paste: true});
+
+    await pageAdvancedModel.general.queryMetadata.selectOption('Annotations');
+
+    await t.click(pageAdvancedConfiguration.saveButton);
+};
+
+test.meta({
+    testPlan: 'https://ggsystems.atlassian.net/browse/GG-26393',
+    testCase: '1'
+})('Validation of user metrics data', async(t) => {
+    await t.resizeWindow(1920, 1080);
+
+    const page = new PageConfigurationBasic();
+
+    await scrollToPageBottom();
+    await page.cachesList.addItem();
+    await scrollToPageBottom();
+    await page.cachesList.addItem();
+
+    await page.saveWithoutDownload();
+
+    await t.click(advancedNavButton);
+    await _createBaseModel(t, 'test.Key1', 'test.Value1');
+    await _createBaseModel(t, 'test.Key2', 'test.Value2');
+    await _createBaseModel(t, 'test.Key3', 'test.Value3');
+
+    await pageAdvancedConfiguration.save();
+
+    await t.navigateTo(resolveUrl('/settings/profile'));
+
+    const newFirstName = 'newFirstName';
+    const newLastName = 'newLastName';
+    const newCompaty = 'newcompaty';
+    const newCountry = 'United States';
+
+    await t
+        .typeText(pageProfile.firstName.control, newFirstName, {replace: true, paste: true})
+        .typeText(pageProfile.lastName.control, newLastName, {replace: true, paste: true})
+        .typeText(pageProfile.company.control, newCompaty, {replace: true, paste: true});
+    await pageProfile.country.selectOption(newCountry);
+    await t.click(pageProfile.saveChangesButton)
+        .expect(successNotification.withText('Profile saved.').exists).ok();
+
+    await t.navigateTo(resolveUrl('/settings/admin'));
+
+    const emailFilterSelector = pageListOfUsers.usersTable.findFilter('Email');
+    const emailCellSelector = pageListOfUsers.usersTable.findCell(0, 'Email');
+
+    await t.typeText(emailFilterSelector, t.ctx.email, {paste: true})
+        .expect(pageListOfUsers.usersTable.findCell(0, 'User').textContent).contains(`${newFirstName} ${newLastName}`)
+        .expect(emailCellSelector.textContent).contains(t.ctx.email)
+        .expect(pageListOfUsers.usersTable.findCell(0, 'Company').textContent).contains(newCompaty)
+        .expect(pageListOfUsers.usersTable.findCell(0, 'Country').textContent).contains('USA');
```

Comment:
Why do we check with `contains`, not `equals`?

Reply:
@MaxLosevskoy `equals` will most likely contain extra characters like newlines and spaces.